sfdoc/publish/tests/test_integration.py

```diff
@@ -675,6 +676,14 @@ def test_same_bundle_push_and_publish_twice(self):
         else:
             tasks.publish_drafts(bundle_A2.pk)  # simulate publish from UI
 
+    @override_settings(REPUBLISH_UNCHANGED_ARTICLES='True')
+    def test_same_bundle_force_republish(self):
+        """Allow publishing identical bundle twice in a row if settings allow."""
+        bundle_A = self.process_bundle_from_webhook(fake_easydita.fake_webhook_body_doc_A)
+        tasks.publish_drafts(bundle_A.pk)  # simulate publish from UI
+        bundle_A2 = self.process_bundle_from_webhook(fake_easydita.fake_webhook_body_doc_A)
+        tasks.publish_drafts(bundle_A2.pk)  # simulate publish from UI
```

Comment:
Does this test actually verify anything, given that it doesn't make an assertion?

Reply:
Clarified in f994ed5eb5ba967ff1ba3da35aba51af0942f3dc